dev/doc/release-process.md

```diff
@@ -20,9 +20,9 @@
 ## On the branching date ##
 
 - [ ] Merge the above PR and create the `vX.X` branch from the last merge commit before this one (using this name will ensure that the branch will be automatically protected).
-- [ ] Set the next major version alpha tag using `git tag -s`.  The `VY.Y+alpha` tag marks the first commit to be in `master` and not in the `vX.X` release branch. Note that this commit is the first commit in the first PR merged in master, not the merge commit for that PR. Therefore, if you proceeded as described above, this should be the commit updating the version, magic numbers and compatibility infrastructure.  After tagging, double-check that `git describe` picks up the tag you just made (if not, you tagged the wrong commit).
+- [ ] Set the next major version alpha tag using `git tag -s` (you can leave the tag message empty).  The `VY.Y+alpha` tag marks the first commit to be in `master` and not in the `vX.X` release branch (be careful about small `v` for branches and big `V` for tags). Note that this commit is the first commit in the first PR merged in master, not the merge commit for that PR. Therefore, if you proceeded as described above, this should be the commit updating the version, magic numbers and compatibility infrastructure.  After tagging, double-check that `git describe` picks up the tag you just made (if not, you tagged the wrong commit).
 - [ ] Push the new tag with `git push upstream VY.Y+alpha --dry-run` (remove the `--dry-run` and redo if everything looks OK).
-- [ ] Start a new ["classic" project](https://github.com/coq/coq/projects?type=classic) to track PR backporting. The project should have a `Request X.X+rc1 inclusion` column for the PRs that were merged in `master` that are to be considered for backporting, and a `Shipped in X.X+rc1` columns to put what was backported. A message to `@coqbot` in the milestone description tells it to automatically add merged PRs to the `Request ... inclusion` column and backported PRs to the `Shipped in ...` column. See previous milestones for examples. When moving to the next milestone (e.g. `X.X.0`), you can clear and remove the `Request X.X+rc1 inclusion` column and create new `Request X.X.0 inclusion` and `Shipped in X.X.0` columns.
+- [ ] Start a new [project](https://github.com/orgs/coq/projects/11/views/4) to track PR backporting. The project should have a `Request X.X+rc1 inclusion` column for the PRs that were merged in `master` that are to be considered for backporting, and a `Shipped in X.X+rc1` columns to put what was backported. A message to `@coqbot` in the milestone description tells it to automatically add merged PRs to the `Request ... inclusion` column and backported PRs to the `Shipped in ...` column. See previous milestones for examples. When moving to the next milestone (e.g. `X.X.0`), you can clear and remove the `Request X.X+rc1 inclusion` column and create new `Request X.X.0 inclusion` and `Shipped in X.X.0` columns.
```

Comment:
this links to "8.20 board" but how is the new project created? Is it supposed to be a separate project from the current "release management" project (projects/11) or is it supposed to be some sub-thing of the "release management" project?

Reply:
Good question, no idea, @Zimmi48 ?

Reply:
Sorry, I should not have waited so long to document this. The idea is to use a single release management project from now on.

The process is the following:

- Create a milestone.
- Add a description saying to which branch(es) its merged PRs should get backported. A simple case is `@coqbot: backport to v8.20 (move rejected PRs to: https://github.com/coq/coq/milestone/60).` A more complex case would look like `@coqbot: backport to v8.19 (move rejected PRs to: <url of current milestone for 8.20>); backport to v8.20 (move rejected PRs to: <URL of current milestone for 8.21>); ...`.
- As soon as a PR is merged in this milestone, the project field for its backporting status will be automatically created by coqbot (with standard values "Request inclusion", "Shipped" and "Rejected").
- You can use pre-existing views in the project to track PR backporting or create new views for new major releases (on the model of previous releases). This could also be automatically done by coqbot, but it isn't the case yet.
- If a PR status is changed to rejected without the milestone being changed first, then coqbot posts a comment to the PR and automatically changes the milestone to the "move rejected PRs to" milestone value.

Reply:
Thanks @Zimmi48 I tried to translate that in the PR, feel free to criticize